common/services/prismic/opening-times.ts

```diff
@@ -219,8 +228,8 @@ export function groupConsecutiveExceptionalDays(
 export function getUpcomingExceptionalPeriods(
   exceptionalPeriods: ExceptionalOpeningHoursDay[][]
 ): ExceptionalOpeningHoursDay[][] {
-  const startOfToday = london().startOf('day').toDate();
-  const upcomingUntil = london().add(28, 'day').endOf('day').toDate();
+  const startOfToday = startOfDay(today());
```

Comment:
Is this something that should be timezoned?

Reply:
Honestly not sure. 🤔 

Currently `startOfDay()` gets the start of the UTC day, so this might go a bit weird around midnight – but at best I'm inclined to leave this until we see issues. (How many people are checking today's opening hours at midnight?)

If the current behaviour is correct, I think it's by intention rather than coincidence. I've gone for the simplest implementation for now, and if that turns out to be wrong we can come back and test it more thoroughly where it breaks.

Reply:
I just was unsure if it would take the time of day of the user, but just tested and looked more into it and looks good :)